src/docs/product/discover-queries/query-builder/query-equations.mdx

```diff
@@ -0,0 +1,141 @@
+---
+title: "Adding Query Equations"
+sidebar_order: 10
+---
+
+<Note>
+
+This feature is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony.
+
+If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.
+
+</Note>
+
+In **Discover**, you can create equations based on the columns of your query. Each column acts like a variable since it represents the result of the query, which is a number. This number is used in the equation.
```

Comment:
```suggestion
In **Discover**, you can add equations based on numeric columns calculated in your query. These columns can be used as input variables into your equation.
```

Reply:
@wmak - let's discuss this edit; i want to make sure we keep this language really simple, but maybe there's a middle ground between the suggestion from @shruthilayaj and our original text.

Reply:
```suggestion
In **Discover**, you can add equations based on your query columns. Each column acts like a variable since it represents the numeric result of the query, and this number is used in the equation.
```